app/services/hawkular_proxy_service.rb

```diff
@@ -8,10 +8,16 @@ def initialize(provider_id, controller)
     @params = controller.params
     @ems = ManageIQ::Providers::ContainerManager.find(@provider_id) unless @provider_id.blank?
     @tenant = @params['tenant'] || '_system'
+
+    @cli = ManageIQ::Providers::Kubernetes::ContainerManager::MetricsCapture::HawkularClient.new(@ems, @tenant)
   end
 
   def client
```

Comment:
@simon3z the `client` method is used in `metric_definitions` and `get_data`

https://github.com/ManageIQ/manageiq-ui-classic/pull/30/files#diff-deb60de663c529247f745d1a17a8e551R49
and in
https://github.com/ManageIQ/manageiq-ui-classic/pull/30/files#diff-deb60de663c529247f745d1a17a8e551R66

without it, I will have to add two `if .. else` that will not add to the readability of this functions.